frontend/projects/upgrade/src/app/core/experiment-design-stepper/experiment-design-stepper.service.ts

```diff
@@ -75,6 +75,10 @@ export class ExperimentDesignStepperService {
   );
   factorialExperimentDesignData$ = this.store$.pipe(select(selectFactorialDesignData), distinctUntilChanged(isEqual));
 
+  // Name:
+  private experimentNameSource = new BehaviorSubject<string>('');
+  currentExperimentName$ = this.experimentNameSource.asObservable();
```

Comment:
I'm not sure what the idea is with this "asObservable" line, I see it sprinkled around our codebase but BehaviorSubjects are observables already, one should be able to use the BehaviorSubject directly and not have to create a second observable (which is also still a behoaviorsubject that you can do .next and .value on directly)